mne/io/artemis123/utils.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import os.path as op
+import inspect
+from ...utils import logger
+
+
+def _load_mne_locs():
+    """Load MNE locs stucture from file (if exists) or recreate it."""
+    # find input file
+    FILE = inspect.getfile(inspect.currentframe())
+    resource_dir = op.join(op.dirname(op.abspath(FILE)), 'resources')
+    loc_fname = op.join(resource_dir, 'Artemis123_mneLoc.csv')
+    if op.exists(loc_fname):
+        logger.info('Loading precomputed mne loc file...')
+        locs = dict()
+        with open(loc_fname, 'r') as fid:
+            for line in fid:
+                vals = line.strip().split(',')
+                locs[vals[0]] = np.array(vals[1::], np.float)
+
+    else:
+        logger.info('Converting Tristan coil file to mne loc file...')
```

Comment:
Let's not have this in here... we don't want (for various packaging and sanity reasons) to write to the MNE install directory. Let's just distribute it, and leave this generation stuff in here as a comment if you want to keep it, or convert it to a test if you really want to be thorough.